packages/starlight/translations/ar.json

```diff
@@ -13,17 +13,17 @@
   "sidebarNav.accessibleLabel": "الرئيسية",
   "tableOfContents.onThisPage": "على هذه الصفحة",
   "tableOfContents.overview": "نظرة عامة",
-  "i18n.untranslatedContent": "هذا المحتوى لا يتوفر بلغتك بعد.",
+  "i18n.untranslatedContent": "هذا المحتوى ليس متوفرا بلغتك بعد.",
   "page.editLink": "عدل الصفحة",
-  "page.lastUpdated": "اخر تحديث:",
+  "page.lastUpdated": "آخر تحديث:",
   "page.previousLink": "السابق",
   "page.nextLink": "التالي",
-  "page.draft": "This content is a draft and will not be included in production builds.",
+  "page.draft": "هذا المحتوى مجرد مسودة ولن يظهر في بنيات الإنتاج",
   "404.text": "الصفحة غير موجودة. تأكد من الرابط أو ابحث بإستعمال شريط البحث.",
   "aside.note": "ملحوظة",
   "aside.tip": "نصيحة",
   "aside.caution": "تنبيه",
   "aside.danger": "تحذير",
   "fileTree.directory": "Directory",
-  "builtWithStarlight.label": "Built with Starlight"
+  "builtWithStarlight.label": "تم بناء الموقع بواسطة Starlight"
```

Comment:
Just to double check — a shorter version of this doesn’t work right? For example, something like:
```suggestion
  "builtWithStarlight.label": "بنيت مع Starlight"
```

The idea of this string is to be a small “Built with Starlight” badge in the page footer, so it can be a bit more “marketing style” than a full sentence. But happy to trust what you think looks best!

Reply:
طوِّر بواسطة Starlight

i think this is the shortest version i can come up with

Reply:
Sounds good to me!